### PR TITLE
Fix#2184, mitmweb -n dispalys incorrect message.

### PR DESCRIPTION
**Pull request by MatthewShao**
Check if the `option.server` is on before printing the information about the proxy server.
In mitmweb, add `server` to `state.setting`, so we could display the correct message in the termstatus.
default

### CODE REVIEW

**Comment by mhils** [comment]
Thanks. I may not have been clear in #2184 - we should re-use the [termstatus addon](https://github.com/mitmproxy/mitmproxy/blob/master/mitmproxy/addons/termstatus.py) for mitmweb. Also updating the web ui sounds good, but it's currently inconsistent with `mitmproxy`, which just hides the `[*:8080]` when `-n` is passed. I suggest we do the same for mitmweb. 😃

**Comment by MatthewShao** [comment]
Sorry for the misunderstanding, no more `[*:8080]` when `-n` is passed.
default

File: web/src/js/components/Footer.jsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ function Footer({ settings }) {
             )}
             <div className="pull-right">
             <span className="label label-primary" title="HTTP Proxy Server Address">
```
Comment (mhils) [inline web/src/js/components/Footer.jsx]: You can make this 
```javascript
{ server && (
    <span className="label label-primary">...</span>
)}
```
as with the other options (see above)

**Comment by MatthewShao** [comment]
Yes, that's better. I haven't noticed this notation before, thanks for the advice.

**Comment by mhils** [comment]
👍

**Pull request merged**